documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -360,6 +360,11 @@ Let's look at what a _create_ event value might look like for our `customers` ta
             "type": "string",
             "optional": true,
             "field": "commit_lsn"
+          },
+          {
+            "type": "int64",
+            "optional": true,
+            "field": "event_serial_no"
```

Comment:
Strictly speaking, the doc change should go into a separate PR, targeting the `development_docs` branch. But given we're going to release the change later this week, I'm fine with leaving it here for this time.